libs/dal/src/repositories/job/job.schema.ts

```diff
@@ -128,15 +128,12 @@ const jobSchema = new Schema<JobDBModel>(
     actorId: {
       type: Schema.Types.String,
     },
-    expireAt: Schema.Types.Date,
     stepOutput: Schema.Types.Mixed,
     preferences: Schema.Types.Mixed,
   },
   schemaOptions
 );
 
-jobSchema.index({ expireAt: 1 }, getTTLOptions());
```

Comment:
Same applies here, remove the index after release to Prod.

Reply:
Yep